php/solutions-manager.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+namespace Ponce_Admin;
+
+use Ponce_Admin\Solutions\Ponce_Logo;
```

Comment:
Wtf php con sus barras inversas. También lo noté en el curso de POO. Me perturba